compiler/locop/src/FormattedTensorShape.test.cpp

```diff
@@ -33,3 +33,15 @@ TEST(FormattedTensorShapeTest, BracketFormat)
 
   SUCCEED();
 }
+
+TEST(FormattedTensorShapeTest, PlainFormat)
+{
+  auto tensor_shape = stdex::make_unique<loco::TensorShape>();
+
+  tensor_shape->rank(2);
+  tensor_shape->dim(0) = 4;
```

Comment:
It doesn't need `tensor_shape->dim(1)`?

Reply:
Ah, copied from existing code and had a missing code!
Thanks!